src/assignments/6-array.js

```diff
@@ -21,3 +28,10 @@ min=1
 
 */
 // start here writing the code for program2
+const ArrayMinMax = [1,4,2,3]
+
+const Max = Math.max(...ArrayMinMax);
+console.log(Max)
+
+const Min = Math.min(...ArrayMinMax);
+console.log(Min)
```

Comment:
Suggest not the use the available JS method here. Write the function that return the min and max number in the array. 
Can get the idea from here and redo the task again: 
https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php